.github/workflows/wheels.yml

```diff
@@ -109,52 +122,48 @@ jobs:
         with:
           fetch-depth: 0
 
-      # TODO: Build wheels from sdist again
-      # There's some sort of weird race condition?
-      # within Github that makes the sdist be missing files
-
-      # We need to build wheels from the sdist since the sdist
-      # removes unnecessary files from the release
-      - name: Download sdist (not macOS)
-        #if: ${{ matrix.buildplat[1] != 'macosx_*' }}
+      - name: Download sdist and pandas_tests
         uses: actions/download-artifact@v4
         with:
-          name: sdist
+          name: sdist-and-tests
           path: ./dist
 
-      - name: Output sdist name (macOS)
-        id: save-path
-        shell: bash -el {0}
-        run: echo "sdist_name=$(ls ./dist)" >> "$GITHUB_ENV"
-
-      # Python version used to build sdist doesn't matter
-      # wheel will be built from sdist with the correct version
+      # The sdist will be corrupted by cibuildwheel/macOS sometimes somehow
+      # (I think this is a Github Actions bug)
       - name: Unzip sdist (macOS)
         if: ${{ startsWith(matrix.buildplat[1], 'macosx') }}
         run: |
-          tar -xzf ./dist/${{ env.sdist_name }} -C ./dist
+          tar -xzf ${{ needs.build_sdist.outputs.sdist_file }} -C ./dist
 
-      - name: Output sdist name (macOS)
+      - name: Output new sdist name (macOS)
         id: save-path2
+        if: ${{ matrix.buildplat[1] == 'macosx_*' }}
         shell: bash -el {0}
-        run: echo "sdist_name=$(cd ./dist && ls -d */)" >> "$GITHUB_ENV"
+        run: echo "sdist_name=./dist/$(cd ./dist && ls -d */)" >> "$GITHUB_ENV"
 
       - name: Build normal wheels
         if: ${{ (env.IS_SCHEDULE_DISPATCH != 'true' || env.IS_PUSH == 'true') }}
         uses: pypa/cibuildwheel@v2.17.0
         with:
-         package-dir: ./dist/${{ startsWith(matrix.buildplat[1], 'macosx') && env.sdist_name || needs.build_sdist.outputs.sdist_file }}
+         package-dir: ${{ startsWith(matrix.buildplat[1], 'macosx') && env.sdist_name || needs.build_sdist.outputs.sdist_file }}
         env:
           CIBW_PRERELEASE_PYTHONS: True
           CIBW_BUILD: ${{ matrix.python[0] }}-${{ matrix.buildplat[1] }}
+          # Note: Since the sdist is the project directory, it does not contain the tests

```

Comment:
Is it a guarantee that the mount directory is always /host in the container? Is that from cibuildwheel?

Reply:
Yep.